EDI/Operation/04_ManageLineItemsFileStt.md

```diff
@@ -0,0 +1,185 @@
+    /M_[Message type, for example INVOIC]\_[EDIFACT version number (last three digits)]/ G\_[Message type, for example INVOIC]\_[EDIFACT version number (last three digits)]\_SG[Group number].     
+    For example: /M_INVOIC_07A/G_INVOIC_07A_SG26   
+    Alternatively, you can copy the two parts from the header file message structure.  
+
+    >[Info] Note that the line items (LIN segment) are included in separate groups, such as the SG26 or SG27 group for the INVOIC.
+
+    ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefMessagData.png)
+
+ 
+6. Click the ![Get](../../Assets/Icons/Download.png "[Get]") [GET] button.   
+    The message structure of the SG26/SG27 group is displayed. The segment is highlighted in red because it is still empty.    
+    It starts with the SG26/SG27 element. 
+
+    ![Message](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructureLineItems.png "[Message]")
+
+
+7. If desired, expand the message structure by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button.   
+    The message structure with its single segments is displayed. It starts with the S_LIN segment. The ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button has changed to an ![Expanded](../../Assets/Icons/NodeExpanded.png "[Expanded]") (Expanded) button.
+
+    ![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructureSG26.png "[Message structure]")
+
+8. Click the [SAVE] button in the top right corner.   
+     - The *Exporter definitions* view is displayed.   
+     - If you now open the definition again for editing, both the *Data Sources* tab and the *Mapping settings* tab is additionally displayed. 
+     - In the header definition file settings, you are no longer able to expand the *G_INVOIC_07A_SG26* or *G_INVOIC_07A_SG27* group.
+
+9. If desired, create the data source and define the mapping setting. For detailed information, see [Prepare definition of file settings](./01_ManageDefinitions.md#prepare-definition-of-file-settings).
+
+
+
+## Specify line item number
+
+Start specifying the line item segments by filling out the single elements of the S_LIN (Line item) segment. This procedure shows how to edit the line item number on the invoice, so that each invoice line item gets a unique and ascending number.
+
+#### Prerequisites
+
+- You have expanded the message structure of the line item definition, see [Get line item segments](#get-line-item-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+ ![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructureSG26.png "[Message structure]")
+
+1. Expand the S_LIN segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at the *S_LIN* segment.   
```

Comment:
*S_LIN* segment (und ähnliche) immer kursiv? Vgl. weiter im Satz (button at the *S_LIN* segment) und z.B. *D_1082* row.